packages/transformer-compile-class/src/index.ts

```diff
@@ -34,12 +34,12 @@ export interface CompileClassOptions {
 
 export default function transformerCompileClass(options: CompileClassOptions = {}): SourceCodeTransformer {
   const {
-    trigger = ':uno:',
+    trigger = 'uno',
```

Comment:
Why do we change this? The idea was allowing ppl to have custom trigger other than forcing to use `:xxx:`

Reply:
Maybe we could have the trigger to accept a RegExp and provide the first group to be the generated name?

Reply:
The trigger is still customizable, but will now require wrapping it in `:`. Possible values are: `:uno:`, `:mytrigger:` or `:mytrigger-foo:`. But we could also switch to a Regex like you suggested so that users have to most flexibility.

Reply:
This could be a breaking change and will lose the flexibility of the choices of conventions, that I would like to avoid.

Reply:
Then we have to move to a user-provided Regex. The requirements would be: We should only accept regular expressions with one group. This group will then be used for the custom compile class name. If the group match is empty (undefined), the transformer will use the hashed name like it currently does.

Let me know if this makes sense, and if yes, I will go ahead and include the required changes in this PR.

Reply:
Yeah I think so, we could use a named regex group. As long as we properly document it I think it should be fine.

Reply:
I now converted the trigger to a user-provided Regex. PR is ready for review.